src/Renderer/MaterialLayer.js

```diff
@@ -126,22 +130,30 @@ class MaterialLayer {
             }
         }
         this.level = EMPTY_TEXTURE_ZOOM;
-        this.textures = [];
-        this.offsetScales = [];
+        this.textures.forEach((v, k) => this.textures.set(k, null));
```

Comment:
No `this.textures.clear()` instead ?

Reply:
I would like keep this `Extents` to assign the texture by `Extent`